cfy_manager/utils/files.py

```diff
@@ -207,9 +210,8 @@ def update_yaml_file(yaml_path, user_owner, group_owner, updated_content):
     yaml.default_flow_style = False
     yaml.dump(yaml_content, stream)
     write_to_file(stream.getvalue(), yaml_path)
-    chown(user_owner,
-          group_owner,
-          yaml_path)
+    if user_owner and group_owner:
```

Comment:
This means that we will ignore user_owner if group_owner isn't supplied. Is this intentional?

Reply:
My idea was that only if `user_owner` and `group_owner` are supplied, we'll change the permissions, and I also didn't want to throw an error in case one of them is not specified. Maybe the right behavior would actually be throwing an error in this case?

Reply:
We should probably raise an exception if one is specified and not the other, as it'll indicate a slipup in our code which will not have the desired effect.

Reply:
Makes sense, I'll fix it.